index.d.ts

```diff
@@ -1,23 +1,24 @@
 import * as React from 'react';
+import FocusTrap from './lib/FocusTrap';
```

Comment:
I've just noticed this after merging. I *think* this may need to be `./es/FocusTrap` (`./lib` is not included in the published npm module).

I'll make the change, unless you enlighten me about something I am misunderstanding.

Reply:
I made the import since I had a compiler warning about `FocusTrap` not being imported, but I don't think any real typings would be coming from it (there would need to be a `FocusTrap.d.ts` file to import I suppose

In any case, there's no runtime with these files, it won't end up in generated code, there's no real import. Shouldn't break anything. But it's up to you, obviously :-)

Reply:
FYI currently 1.0.5 is reporting:
```
react-hotkeys/index.d.ts(2,23): error TS7016: Could not find a declaration file for module './cjs/FocusTrap'. './node_modules/react-hotkeys/cjs/FocusTrap.js' implicitly has an 'any' type.
```

Reply:
Thanks for taking the time to report, @mykter. It seems like we can't get away with importing a file that doesn't exist. I'll take a look at this.

Reply:
Any more info about your setup? This installed fine for me locally (`npm pack`ing in this project and installing to another). Note that neither `lib/FocusTrap.js` nor `cjs/FocusTrap.cjs` have any typing info.

I can return to this after the easter weekend

Reply:
This was also reported as a [separate issue](https://github.com/greena13/react-hotkeys/issues/89). Seems to occur when `noImplicityAny` is enabled in `tsconfig.json`. 

I've released [v1.0.6](https://github.com/greena13/react-hotkeys/commit/4478bdbda1b6788395d40f08a94151cc098dacbc), which I hope has resolved the issue.

Reply:
I use "strict": true, which includes noImplicitAny.
1.0.6 does indeed address it :+1:

Reply:
Very odd, I use `noImplicitAny`on the testing project